php/chubbyphp/composer.json

```diff
@@ -0,0 +1,7 @@
+{
+  "require": {
+    "chubbyphp/chubbyphp-framework": "^2.8",
+    "nikic/fast-route": "^1.3",
+    "slim/psr7": "^1.0"
```

Comment:
could you use `*`, adding a constraint for this add more maintenance on this project :stuck_out_tongue:

Reply:
done